app/controllers/hyrax/workflow_actions_controller.rb

```diff
@@ -37,5 +37,13 @@ def after_update_response
           wants.json { render 'hyrax/base/show', status: :ok, location: polymorphic_path([main_app, curation_concern]) }
         end
       end
+
+      def respond_to_html
+        if workflow_action_form.authorized_for_processing
```

Comment:
We shouldn't have authorization logic in this method that is only used for HTML users.   We'd expect the same logic when making a JSON request too, right?

Reply:
Should you call `workflow_action_form.valid?` instead. It looks like `authorized_for_processing` is just one of the validators of the form.

Reply:
I could be wrong, but I believe we drop into this branch when `workflow_action_form.valid?` is already false as a result of `workflow_action_form.save`, so in this check, we're determining if the validation error is authorization based or due to the missing comment. So checking `workflow_action_form.valid?` would be redundant and not provide enough information. Is there another check available that would make more sense in this context?

Reply:
I'm concerned that if  you call this a second time it will add an additional error here: https://github.com/samvera/hyrax/blob/e005a915b3ff56b39bff985885def458f350eb63/app/forms/hyrax/forms/workflow_action_form.rb#L49

Reply:
Is that a significant enough issue to warrant concern? Maybe we can walk through this together at some point soon? I'm a bit stuck in that we need to know if the error is based on authorization or just a missing comment, and I'm not seeing another way to check this without duplication of code.